src/SFA.DAS.EAS.Portal/Database/Models/Account.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Models;
+using SFA.DAS.EAS.Portal.Types;
+
+namespace SFA.DAS.EAS.Portal.Database.Models
+{
+    public class Account : Document, IAccountDto
+    {
+        [JsonProperty("accountId")]
+        public long AccountId { get; private set; }
+
+        [JsonProperty("accountLegalEntities")]
+        public IEnumerable<IAccountLegalEntityDto> AccountLegalEntities => _accountLegalEntities;
+
+        [JsonProperty("organisations")]
+        public ICollection<Organisation> Organisations { get; set; }
+
+        [JsonProperty("outboxData")]
+        public IEnumerable<OutboxMessage> OutboxData => _outboxData;
```

Comment:
This should be on a separate interface.  It's not part of the Account read model.  Same for the created/updated/deleted etag etc.. fields.

Reply:
OutboxData is not part of the interface, it only exists in the concrete class.

Created and updated are also not part of the interface. Deleted is, as it is both part of the write model and DTO, so exists in the interface to keep the two in sync.

Reply:
OutboxData is no longer part of Account.